src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="bar"
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <label>
+        <input
+          class="search"
+          type="text"
+          name="Bigbar"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+      <img
+        class="icon"
+        src="images/Search.svg"
+        alt="search"
+      >
+    </form>
+    <form
+      class="bar"
+      action="#"
+      method="post"
+      data-qa="small"
+    >
+      <label>
+        <input
+          class="search search_small"
+          type="text"
+          name="Smallbar"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+      <img
+        class="icon icon_small"
+        src="images/Search.svg"
+        alt="search"
+      >
+    </form>
   </body>
-</html>
+</html>
```

Comment:
```suggestion
</html>

```
add empty line in the end of the file